Assets/_Scripts/Player/Modules/SceneGraph/UI/SayThinkBubble.cs

```diff
@@ -54,6 +54,7 @@ void Start()
         }
 
         public void SetText(string text, UnityEngine.Color c, TMP_FontAsset font, float scale){
+            text = System.Text.RegularExpressions.Regex.Unescape(text);// Converts escape sequences in strings into the characters they represent
```

Comment:
I have a few, very pedantic comments.
Please put the comment above the code on its own line. Just a personal preference, yes, but it is much more common in our code base, and makes for shorter lines.

Generally good to avoid overwriting the value in an argument if it is not needed, and here you can simply assign the unescaped text to `bubbleText.text`.